### DISCUSSION TITLE
Copilot in Neovim with LSP (<Tab> already used)

### DISCUSSION THREAD

**Discussion by matisla** [community/community#43569] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi,

How could I use Copilot in Neovim in parallel with auto-completion tools working with LSP.
I already use auto-completion with <Tab> and I would like to use Copilot with <C-Space>. How could I configure it in my configuration file (in best case in Lua).

**Comment by jroes** [comment]
I think this should help: https://github.com/community/community/discussions/40858#discussioncomment-4620599
